server/routers/work-shares/activateWallet.ts

```diff
@@ -30,9 +30,31 @@ export const activateWallet = protectedProcedure
     const deviceAndLocationIdPromise = getDeviceAndLocationId(ctx);
     const now = Date.now();
 
+    // Explicitly set JWT claims before queries
+    try {
+      // Set JWT claims with session-level SET
+      const jwtClaims = JSON.stringify({
```

Comment:
Based on the latest comments, this is already included in `ensureJwtClaimsSet`, right?